dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -171,13 +174,13 @@ static StaticLinkageChecker getInstanceFromArguments(String[] arguments) throws
       boolean reportOnlyReachable = cmd.hasOption("r");
 
       if (jarFilePaths.isEmpty()) {
-        System.err.println("No jar files to scan");
+        logger.severe("No jar files to scan");
```

Comment:
This probably should be System.err.println since this is the UI

Reply:
Removed the message as the exception in next line delivers the message.